src/generator/AutoRest.CSharp.Azure/TemplateModels/AzureMethodTemplateModel.cs

```diff
@@ -278,14 +278,14 @@ public override string BuildUrl(string variableName)
 
         private void AddQueryParametersToUri(string variableName, IndentedStringBuilder builder)
         {
-            builder.AppendLine("List<string> _queryParameters = new List<string>();");
+            builder.AppendLine("System.Collections.Generic.List<string> _queryParameters = new System.Collections.Generic.List<string>();");
```

Comment:
If we use `var` for declarations wherever we can in places like this, 

``` csharp
   builder.AppendLine("var _queryParameters = new System.Collections.Generic.List<string>();");
```

we'll have less breakage later on if we change the type (and it cuts down on redundancy).